wordpress-mobile/WordPress-iOS
# Allow changing site in Post editor when current site is hidden

Fixes #5296 

If the current posting site is hidden, previousIndexPath will be `nil` causing `[self cancelButtonTapped:nil]` to be called. Now we check to make sure preiviousIndexPath is not nil.

To test:
1. Start a post from a hidden site
2. Change site using the site picker on the top left
3. Verify site is changed

Needs review: @diegoreymendez
